skyline-vscode/react-ui/src/data/mock_data.js

```diff
@@ -883,94 +883,100 @@ export const profiling_data = {
   ],
   energy: {
     current: {
-      total_consumption: 4.8,
-      components: [
-        { type: "ENERGY_UNSPECIFIED", consumption: 0 },
-        { type: "ENERGY_CPU_DRAM", consumption: 2.1 },
-        { type: "ENERGY_GPU", consumption: 2.7 },
-      ],
-    },
-    past_measurements: [
-      {
-        total_consumption: 5.10,
-        components: [
-          { type: "ENERGY_UNSPECIFIED", consumption: 0 },
-          { type: "ENERGY_CPU_DRAM", consumption: 3.05 },
-          { type: "ENERGY_GPU", consumption: 2.05 },
-        ],
-      },
-      {
-        total_consumption: 7.30,
-        components: [
-          { type: "ENERGY_UNSPECIFIED", consumption: 0 },
-          { type: "ENERGY_CPU_DRAM", consumption: 2.15 },
-          { type: "ENERGY_GPU", consumption: 5.15 },
-        ],
-      },
-      {
-        total_consumption: 8.20,
+      // total_consumption: 4.8,
```

Comment:
Delete commented lines

Reply:
Sorry, in previous commit I commented for testing purposes, now all is uncommented